src/ReflectionParameter.php

```diff
@@ -24,6 +29,11 @@ class ReflectionParameter implements \Reflector
     private $isOptional;
 
     /**
+     * @var bool
+     */
+    private $hasDefaultValue;
```

Comment:
Don't need a default value, we can only instantiate `ReflectionParameter::createFromNode` which always sets `$hasDefaultValue` value